src/content/en/fundamentals/accessibility/02-semantics-bultin/sembuiltin-three.md

```diff
@@ -0,0 +1,62 @@
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+When we use a man-made tool or device, we typically look to its form and design to give us an idea of what it does and how it works. An *affordance* is any object that offers, or affords, its user the opportunity to perform an action. The better the affordance is designed, the more obvious or intuitive its use.
+
+Some affordances can be misleading. A common example is a door with a U-shaped handle that most users expect to pull, but that only responds to a push. Other affordances, such as a door with a flat plate that is clearly meant to be pushed, are more direct and obvious.
+
+In graphical user interfaces, affordances represent actions we can take, but they can be ambiguous because there is no physical object to interact with. GUI affordances are thus specifically designed to be unambiguous: buttons, check boxes, and scroll bars are meant to convey their usage with as little training as possible.
+
+For example, you might paraphrase the use of some common form elements (affordances) like this:
+
+ - Radio buttons &mdash; "I can choose one of these options."
+ - Check box &mdash; "I can choose 'yes' or 'no' to this option."
+ - Text field &mdash; "I can type something into this area."
+ - Dropdown &mdash; "I can open this element to display my options."
+
+You are able to draw conclusions about these elements *only because you can see them*. Naturally, someone who can't see the visual clues provided by an element can't comprehend its meaning or intuitively grasp the value of the affordance. So we must make sure that the information is expressed flexibly enough to be accessed by assistive technology that can construct an alternative interface to suit its user's needs.
+
+This non-visual exposure of an affordance's use is called its *semantics*. The WebAIM WCAG guidelines address it like this.
+
+![guideline4-1.png](imgs/guideline4-1.png)
+
+Let's try out some things from the point of view of someone using assistive technology. In the introduction, you had a brief look at ChromeVox Lite. Now we'll try using it without any access to the web page's visible interface.
+
+We've modified the flight reservation page from the previous lesson so that you can't see what you're doing any more, and you'll have to use the screen reader. You might want to refer back to the earlier exercise if you get stuck. As before, use the form to book a one-way ticket to Melbourne with a window seat.
+
+<a href="http://robdodson.github.io/udacity-a11y/lesson3-semantics-built-in/02-chromevox-lite/" target="_blank">ChromeVox lite demo page</a>
```

Comment:
Use this url http://udacity.github.io/ud891/lesson3-semantics-built-in/02-chromevox-lite/